templates/docs/patterns/split-list/index.md

```diff
@@ -0,0 +1,75 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Split list | Patterns
+---
+
+The split list pattern is used to easily implement a list of paired titles and
+descriptions, complete with a top-level title and description. An optional CTA
+block is placed at the bottom of the content as well. Depending on the size and
+composition of your content, you can choose from a variety of split list
+layouts.
+
+## 50/50 on desktop w/ description
+
+This variant contains a top-level description which is presented side-by-side
+with its title on desktop screen sizes.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/split-list/50-50-desktop-with-description/" class="js-example">
+View example of the matrix pattern
+</a></div>
+
+## 50/50 on tablet w/o description
+
+This variant does not contain a top-level description and its child list is
+presented with its titles side-by-side with its descriptions on tablet screen
+sizes.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/split-list/50-50-tablet-no-description/" class="js-example">
```

Comment:
Not a show-stopper, and not sure if this was a stylistic choice or not, but there are some places throughout these examples that could use a bit more whitespace control to get rid of extra newlines
![image](https://github.com/user-attachments/assets/bb179786-c68a-46a8-a559-22096de5aa0d)